snyk-monitor/templates/pvc.yaml

```diff
@@ -0,0 +1,13 @@
+{{ if .Values.pvc.enabled }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ .Values.pvc.name }}
+spec:
+  accessModes:
+    - ReadWriteOnce
```

Comment:
As mentioned in Slack, there's a tradeoff here. To support a large number of storage drivers (including EBS which our customer uses), we specify ReadWriteOnce, which only supports access from a single pod. This _may_ make it harder to horizontally scale snyk-monitor to multiple pods in future